src/GraphQL/Introspection/__Type.cs

```diff
@@ -38,7 +38,7 @@
 
             Field<StringGraphType>("name").Resolve(context => context.Source!.Name);
 
-            Field<StringGraphType>("description");
+            Field<StringGraphType>("description").Resolve(context => context.Source!.Description);
```

Comment:
This is one such change you mentioned.

Reply:
What change? Refactor code to use types/members at compile-time instead of run-time to expose the whole picture to the trimmer? If so, OK, good example eliminating `NameFieldResolver` .